app/views/posts/_comment_form.html.erb

```diff
@@ -0,0 +1,13 @@
+<% if current_page?(edit_comment_path)%>
+<h3>Edit this comment:</h3>
+<% else %>
+<h3>Add new comment:</h3>
+<% end %>
+
+
+
```

Comment:
- submitting a blank comment blows up!. please handle the form errors returned.
![Screenshot 2019-09-02 at 4 17 07 PM](https://user-images.githubusercontent.com/10922508/64123773-2cff5000-cd9d-11e9-91ca-01f5e1a03a63.png)